tithiwa/group.py

```diff
@@ -240,6 +240,14 @@ def _wait_for_group_info_to_load(self):
                 chatinfo = self._wait_for_presence_of_an_element(SELECTORS.CHATROOM__INFO).get_attribute('innerText')
             except:
                 pass
+    def join_group(self,url):
+        code_pos = url.index('com/')
+        code_pos += 4
+        code = url[code_pos:]
+        group_url = 'https://web.whatsapp.com/accept?code='+ code
+        driver.get(group_url)
+        time.sleep(10)
```

Comment:
using `sleep()` may create problems, I explained why we should avoid sleep() at this [PR](https://github.com/Tithibots/tithiwa/pull/63#issuecomment-714571161)

Reply:
What would be the alternative then?

Reply:
by using `self._wait_for_presence_of_an_element(selector)`